src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -91,7 +91,13 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     };
   }
 
-  writeDrupalIndexPage(files);
+  writeDrupalDebugPage(files);
+  files[`drupal/index.md`] = {
```

Comment:
Temporary index that is tagged as a drupal page, so it has links pointing to drupal pages when they exist.